app/models/admin.rb

```diff
@@ -1,18 +1,48 @@
 class Admin < ActiveRecord::Base
   include ActiveModel::ForbiddenAttributesProtection
-  # Authlogic gem
-  acts_as_authentic do |config|
-    config.transition_from_restful_authentication = true
-    if (ArchiveConfig.BCRYPT  || "true")  == "true" then
-      config.crypto_provider = Authlogic::CryptoProviders::BCrypt
-      config.transition_from_crypto_providers = [Authlogic::CryptoProviders::Sha512, Authlogic::CryptoProviders::Sha1]
-    else
-      config.crypto_provider = Authlogic::CryptoProviders::Sha512
-      config.transition_from_crypto_providers = [Authlogic::CryptoProviders::Sha1]
+
+  devise :database_authenticatable,
+         :registerable,
```

Comment:
I don't know if we want the :rememberable as we want admins to have to log in each time. I can also see an issue later when we would want Users to be able to stay logged in ( probably for a different amount of time that we might want to keep admins logged in for ).

I will remove registerable thank you.